src/main/resources/cli-help/ballerina-dist-pull.help

```diff
@@ -1,13 +1,12 @@
 NAME
-       ballerina-dist-pull - Fetch a given distribution and set it as the active version
+       ballerina-dist-pull - Fetch a given distribution
```

Comment:
AFAIK, `ballerina dist pull` does what is said here. It both pulls the given distribution and sets it as the current version. Was your experience different than what is described in here and in docs?

Reply:
No, it works as described but it is really annoying that it automatically sets the pulled version. Lets say I want to pull 3 different versions. Current pull command changes my active version 3 times and if I want to keep using my old version I have to change it manually again. Maybe we should have a download option but since we have use command I thought it is unnecessary to add a new command.

Reply:
IINM there's a proposed `ballerina dist fetch` command which would do just the same, i.e., just fetch the specified version but not set it as the active distribution.

`dist pull` is just as a shortcut for `dist fetch` + `dist use`

Reply:
fetch command is what I want but I thought since we have two commands that can achieve what I want modifying one is better than adding another command and increasing complexity